composer.json

```diff
@@ -184,6 +185,9 @@
             "@composer create-test-db",
             "@php -d date.timezone=UTC ./vendor/bin/phpunit -c tests-legacy/old-controllers.xml"
         ],
+        "integration-domain-tests": [
```

Comment:
this could be named "behavior-driven-tests", since we will use it for unit/integration/e2e tests